apps/docs/src/app/core/component-docs/datetime-picker/examples/datetime-format-example/datetime-format-example.component.html

```diff
@@ -5,4 +5,4 @@
 <br />
 <br />
 <br />
-Selected: {{ date || 'null' }}
\ No newline at end of file
+Selected: {{ date ? (date | dateTimeFormatPipe) : 'null' }}
```

Comment:
You can use optional chaining for this, and cover the scenario in the pipe

Reply:
In my opinion it is better to allow the application developer to handle this themselves, maybe they want to display "null" or maybe "no date provided" etc.